style/stylesheets/betterttv-dark.css

```diff
@@ -2384,3 +2384,14 @@ span.subscribe-text {
 .chatReplay .loading-spinner-container {
     background: rgba(10,10,10,.85);
 }
+
+/* host & playlist banner fix */
+.player-column .brick, .host-frame .brick {
+	background-color: #222;
+	border-left: 5px solid #6441a5 !important;
+	box-shadow: 0px 0 0 #6441a4 inset !important;
+}
+
+.what {
```

Comment:
Apparently the background color of the dark mode banner was [fixed](https://github.com/night/BetterTTV/commit/d20732c328e68d1abcb03e16ca46959128fafc1c) yesterday by @teak, I'll update this PR with just the purple bar and button coloring.